docs/extensions/logging-and-monitoring.md

```diff
@@ -77,7 +77,7 @@ spec:
 
 #### Aggregate Prometheus
 
-Another Prometheus instance in the `garden` namespace (called "aggregate Prometheus") stores pre-aggregated data from the cache Prometheus and shoot Prometheis.
+Another Prometheus instance in the `garden` namespace (called "aggregate Prometheus") stores pre-aggregated data from the cache Prometheus and shoot Prometheus.
```

Comment:
FYI, this is not a typo, Prometheus in plural is Prometheis: https://prometheus.io/docs/introduction/faq/#what-is-the-plural-of-prometheus

Reply:
I never thought that about this, thanks for sharing!
In this case, I think the singular form should be used (we have one prometheus per shoot), so the change is correct, right?